src/pkg/transform/git_test.go

```diff
@@ -48,23 +48,23 @@ func TestMutateGitURLsInText(t *testing.T) {
 	originalText := `
 	# Here we handle invalid URLs (see below comment)
 	# We transform https://*/*.git URLs
-	https://github.com/defenseunicorns/zarf.git
+	https://github.com/zarf-dev/zarf.git
 	# Even URLs with things on either side
-	stuff https://github.com/defenseunicorns/zarf.git andthings
+	stuff https://github.com/zarf-dev/zarf.git andthings
 	# Including ssh://*/*.git URLs
-	ssh://git@github.com/defenseunicorns/zarf.git
+	ssh://git@github.com/zarf-dev/zarf.git
 	# Or non .git URLs
 	https://www.defenseunicorns.com/
 	`
 
 	expectedText := `
 	# Here we handle invalid URLs (see below comment)
 	# We transform https://*/*.git URLs
-	https://gitlab.com/repo-owner/zarf-1211668992.git
+	https://gitlab.com/repo-owner/zarf-4156197301.git
 	# Even URLs with things on either side
-	stuff https://gitlab.com/repo-owner/zarf-1211668992.git andthings
+	stuff https://gitlab.com/repo-owner/zarf-4156197301.git andthings
 	# Including ssh://*/*.git URLs
-	https://gitlab.com/repo-owner/zarf-2566185087.git
```

Comment:
Why do these change?

Reply:
`MutateGitURLsInText` appends a crc32 hash of the repo name to the end of the URL. The repo name changed so the hash changed

https://github.com/zarf-dev/zarf/blob/4bd52447194a771d38f6694027d90b6ee8c4aa2b/src/pkg/transform/git.go#L81-L86

Reply:
Makes sense in that case